app/scripts/modules/core/.npmignore

```diff
@@ -0,0 +1,5 @@
+yalc.*
+.*
```

Comment:
@anotherchrisberry @jrsquared FYI, this excludes .happypack and .awcache from the npm package